tbump.toml

```diff
@@ -40,3 +40,11 @@ src = "CITATION.cff"
 name = "candidate"
 # the default value to use, if there is no match
 default = ""
+
+[[file]]
+src = "Cargo.toml"
+search = '^version = "{current_version}"'
+
+[[file]]
+src = "docs/advanced/github_actions.md"
+search = "pixi-version: v{current_version}"[[field]]
```

Comment:
small question: do we need to have this [[field]] at the end of tbump.toml?

Reply:
Yeah sorry removed it